packages/matchbox/src/components/Modal/tests/__snapshots__/Modal.test.js.snap

```diff
@@ -39,7 +39,7 @@ exports[`Modal should render contents when open 1`] = `
 `;
 
 exports[`Modal should render contents when open 2`] = `
-<FocusLockCombination
+<ForwardRef
```

Comment:
Yep, looks like this should be reverted since `FocusLockCombination` is the correct display name